src/plugins/tonegen/envelop.h

```diff
@@ -0,0 +1,67 @@
+/*************************************************************************
+This file is part of tone-generator
+
+Copyright (C) 2010 Nokia Corporation.
+
+This library is free software; you can redistribute
+it and/or modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation
+version 2.1 of the License.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+USA.
+*************************************************************************/
+
+#ifndef __TONEGEND_ENVELOP_H__
+#define __TONEGEND_ENVELOP_H__
+
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
```

Comment:
Wonder if that's really needed? If yes, maybe we can define it in the makefile/automake stuff?